deploy/base/daemonset/swisnap-agent-daemonset.yaml

```diff
@@ -19,21 +21,48 @@ spec:
         effect: NoSchedule
       containers:
       - name: swisnap-agent-ds
-        image: 'solarwinds/solarwinds-snap-agent-docker:latest'
-        imagePullPolicy: Always
+        image: solarwinds/solarwinds-snap-agent-docker:latest
+        imagePullPolicy: IfNotPresent
+        ports:
+          - name: probe
+            containerPort: 21413
+            protocol: TCP
         env:
           - name: APPOPTICS_HOSTNAME
             valueFrom:
               fieldRef:
                 fieldPath: spec.nodeName
           - name: APPOPTICS_TOKEN
             valueFrom:
-              configMapKeyRef:
+              secretKeyRef:
                 name: appoptics-token
                 key: APPOPTICS_TOKEN
-        envFrom:
-          - configMapRef:
-              name: swisnap-agent-ds-configmap
```

Comment:
Note:  configMapRef was replaced via straight env variables, because the pod behavior for some reason isn't the same between the two, though `env | grep SWI` is the same for both implementation.

I've attached the logs files for both implementations.
The env print out comes from me adding `env | grep SWI` to the swisnap-init.sh for debugging purposes when testing these issues

[configMapEnv.log](https://github.com/solarwinds/solarwinds-snap-agent-docker/files/4602393/configMapEnv.log)
[env.log](https://github.com/solarwinds/solarwinds-snap-agent-docker/files/4602394/env.log)

Reply:
can you elaborate what is a difference exactly. It was intentionally changed to config maps some longer time ago and have no issues with it ?

Reply:
See https://github.com/solarwinds/solarwinds-snap-agent-docker/pull/14 for more information and to try to reproduce.

Reply:
I added the configmaps back.  While the logs are definitely producing heartbeat issues, appoptics still has the metrics.
<img width="1417" alt="Screen Shot 2020-05-11 at 5 12 32 AM" src="https://user-images.githubusercontent.com/8380706/81550631-28d0fc80-9346-11ea-9fc5-6907db944132.png">
<img width="1397" alt="Screen Shot 2020-05-11 at 5 12 12 AM" src="https://user-images.githubusercontent.com/8380706/81550646-2ec6dd80-9346-11ea-83bf-0a0c4f6bc8c3.png">
<img width="1434" alt="Screen Shot 2020-05-11 at 5 11 59 AM" src="https://user-images.githubusercontent.com/8380706/81550697-3d14f980-9346-11ea-9e16-00be4fe7a07a.png">
<img width="1420" alt="Screen Shot 2020-05-11 at 5 11 31 AM" src="https://user-images.githubusercontent.com/8380706/81550713-40a88080-9346-11ea-83ec-5615ae8efa7e.png">